### DISCUSSION TITLE
JsPsych Citations

### DISCUSSION THREAD

**Discussion by ghost** [jspsych/jsPsych#2454] [Q&A]
I am planning to adapt the code from the demo-flanker experiment in jsPysch 7 to my own experiment  but I am not sure who to cite. I have already cited (de Leuw, 2015) in my paper. Would the citation be the same for the demo experiments ? Thank you

**Comment by jodeleeuw** [comment]
Hi @Vizitok 

It's fine to cite the 2015 paper for this. Thanks for checking!
